dotcom-rendering/src/types/front.ts

```diff
@@ -248,7 +248,10 @@ export type DCRFrontCard = {
 	webPublicationDate?: string;
 	image?: string;
 	kickerText?: string;
+	supportingContent?: DCRSupportingContent[];
 	snapData?: DCRSnapType;
+	isBoosted?: boolean;
+	isCrossword?: boolean;
```

Comment:
I am a big fan of this change! I've always found this confusing! I have a question regarding the props you chose to add to the `DCRFrontCard`. [`TrailType` was extending `BaseTrailType`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/types/trails.ts#L6-L38) and it had some props coming from its supertype I do not see here, e.g. `ageWarning`, `shortUrl`, `commentCount` etc. Is this intentional?

Another difference I see is `showQuotedHeadline` used to be optional in `TrailType` but not in `DCRFrontCard`.

Reply:
Makes sense, thanks!